deployment/aws/cdk/read.14868.1.lock

```diff
@@ -0,0 +1 @@
+14868
```

Comment:
I'm not sure what is this file and why it has to be included

Reply:
Removed. Not sure what was it either.